src/main/java/nextstep/fp/Lambda.java

```diff
@@ -18,39 +19,28 @@ public static void printAllLambda(List<Integer> numbers) {
     }
 
     public static void runThread() {
-        new Thread(new Runnable() {
-            @Override
-            public void run() {
-                System.out.println("Hello from thread");
-            }
-        }).start();
+        new Thread(() -> System.out.println("Hello from thread")).start();
     }
 
-    public static int sumAll(List<Integer> numbers) {
+    public static int sum(List<Integer> numbers, Conditional<Integer> conditional) {
         int total = 0;
         for (int number : numbers) {
-            total += number;
+            if (conditional.test(number)) {
+                total += number;
+            }
         }
         return total;
```

Comment:
이 부분도 람다로 변경이 가능할 것 같아요 :)
```suggestion
        numbers.stream()
               .filter(conditional::test)
               .reduce(0, Integer::sum);
```